src/Chapter2.hs

```diff
@@ -501,7 +527,10 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+
+isThird42 :: (Eq a, Num a) => [a] -> Bool
+isThird42 (_ : _ : third : _) = third == 42
```

Comment:
You don't need to compare `third` with 42 separately, you can pattern match on `42` directly in the pattern.